dev/com.ibm.ws.classloading/src/com/ibm/ws/classloading/internal/ContainerClassLoader.java

```diff
@@ -357,6 +369,17 @@ public ContainerUniversalResource(Container c) {
             container = c;
         }
 
+        @Override
+        @Trivial
+        public InputStream getInputStream() {
+            URL u = getResourceURL("jar");
```

Comment:
```suggestion
            URL u = getResourceURL("wsjar");
```
- Don't we want wsjar so we get the caching and don't need to read from the entry every single time?  

- Also without updating the StreamHandler implementation we won't get much benefit since this is what it is already doing.  

- Before with the cache of the URL we were not having to create the URL each time.  Now we will have to create the URL each time.